### ISSUE TITLE
[BUG] "Experimental" and Support SLAs

### ISSUE THREAD

**Issue by stockholmux** [opensearch-project/OpenSearch-Dashboards#800]
**Describe the bug**

When visualization and adding a control, there is messaging about support SLAs.

**To Reproduce**
1. Go to Visualize
2. Click on Create New Visualization
3. Hover over the "E" on the controls button. 
4. See the weird message about Support SLAs 
5. Click on controls
6. See weird message "This visualization is experimental and is not subject to the support SLA of official GA features. For feedback, please create an issue in GitHub" 

**Expected behavior**
Support SLAs aren't part of an open source project, messaging that reflects this

**OpenSearch Version**
1.0

**Dashboards Version**
1.0
**Plugins**

All standard distribution plugins

**Screenshots**

<img width="388" alt="Screen Shot 2021-09-16 at 8 21 26 AM" src="https://user-images.githubusercontent.com/1152927/133631759-6efed095-0a61-447d-9298-8d2c1258057b.png">
<img width="1026" alt="Screen Shot 2021-09-16 at 8 21 19 AM" src="https://user-images.githubusercontent.com/1152927/133631776-f2a6b72f-1e29-44ca-939d-581b5a54c8c5.png">

**Additional context**

**Comment by tmarkley** [comment]
Thanks for documenting this Kyle! Do you have recommendations on what the new messaging should be? @aetter may have input?

**Comment by aetter** [comment]
Sure, a lot of it has to do with our attitude towards experimental features going forward. They're not something we've traditionally done, which indicates we should just do quick-fix wording and call it a day. If we plan to include experimental features in our releases with some regularity, though, we can and should put more thought into the wording and aim for more user engagement/feedback.

Quick-fix wording would be something like:

```
Experimental
We might remove or make major changes to this visualization in the future.

This visualization is experimental. To provide feedback on it, please [submit a GitHub issue](https://github.com/opensearch-project/OpenSearch-Dashboards/issues).
```

**Comment by stockholmux** [comment]
I like what @aetter is saying. I would question, though, if this is really experimental from the OpenSearch perspective. Is OpenSearch planning on removing or making major changes? If not, I think it can be trashed all together.

**Comment by aetter** [comment]
Agreed, yeah, if we plan to leave this feature in as-is for an extended period of time (and not really do experimental features going forward), trashing the message altogether also seems like a fine outcome.

**Comment by tmarkley** [comment]
Based on our team conversation, we'll want to remove the Experimental flag on Controls and check what else is marked as Experimental. We may need to address other uses on a case-by-case basis.

**Comment by stockholmux** [comment]
Hooray!

**Comment by kavilla** [comment]
> we'll want to remove the Experimental flag on Controls and check what else is marked as Experimental. We may need to address other uses on a case-by-case basis.

To re-cap, we should have a list of all experimental features, and dive into current experience of experimental features and find out if anything was missing not make it non-experimental. If we do, we should still remove the experimental but address the issues.

**Comment by ashwin-pc** [comment]
Experimental Advanced settings

1. Big warning on top of [Advanced Settings](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/d7004dc5b0392477fdd54ac66b29d231975a173b/src/plugins/advanced_settings/public/management_app/components/call_outs/call_outs.tsx#L54)
2. Experimental settings in advanced settings
    1. [query.allowWildcards](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/d7004dc5b0392477fdd54ac66b29d231975a173b/src/plugins/data/server/ui_settings.ts#L129) 
    2. [graphiteURLDescription](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/vis_type_timeline/server/plugin.ts#L177) 
    3. [quandlKeyDescription](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/vis_type_timeline/server/plugin.ts#L189)
    4. [visualize:EnableLabs](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/visualizations/server/plugin.ts#L71) 

Experimental visualizations

* [Input controls viz](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/d7004dc5b0392477fdd54ac66b29d231975a173b/src/plugins/input_control_vis/public/input_control_vis_type.ts#L54) 
* [Vega Map layer](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/vis_type_vega/public/components/experimental_map_vis_info.tsx#L41) 

Time series datasources

* [Graphite](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/vis_type_timeline/server/series_functions/graphite.js#L58) 
* [Worldbank](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/vis_type_timeline/server/series_functions/worldbank.js#L54) 
* [Worldbank Indicators](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/vis_type_timeline/server/series_functions/worldbank_indicators.js#L63) 
* [Quandl](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/vis_type_timeline/server/series_functions/quandl.js#L60) 

Other mentions of experimental

> These aren't experimental features, but rather provide support and surface messages for experimental features.

* [Experimental visualizations in visualization picker](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/visualizations/public/wizard/new_vis_modal.tsx#L124) 
* [Experimental Viz banner](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/visualize/public/application/components/experimental_vis_info.tsx) 
* [Functional test page objects](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/test/functional/page_objects/visualize_page.ts#L221) 
* [Experimental viz functional tests](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/test/functional/apps/visualize/index.ts#L72) ([test file](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/test/functional/apps/visualize/_experimental_vis.js))
* [Experimental Viz tooltip](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/visualize/public/application/utils/get_table_columns.tsx#L57-L71) 
* [Fetch experimental visualizations](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/visualize/public/application/components/visualize_listing.tsx#L122) 
* [Experimental viz icon CSS](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/visualize/public/application/components/visualize_listing.scss#L12)

**Comment by tmarkley** [comment]
@ahopp from the product side do you have thoughts on the path forward for each of these "experimental" features? We need to decide which of them we should improve, which of them can remain the same with different messaging, and which should be removed.

**Comment by tmarkley** [comment]
Next step: document the state of each of these features individually in order to determine which path to take.

**Comment by boktorbb-amzn** [comment]
I'll start documenting for each experimental feature/setting. Do we have a preference on moving this into a new issue vs continuing in here?

**Comment by tmarkley** [comment]
> I'll start documenting for each experimental feature/setting. Do we have a preference on moving this into a new issue vs continuing in here?

@boktorbb-amzn I'd say that this can be the meta/epic/parent issue and we should edit the description with a list of each of the features with their own issues.

**Comment by joshuarrrr** [comment]
I think we've found that we'll actually want to keep many of the experimental visualization features, as they've been particularly useful in launching drag and drop: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1708

**Comment by seanneumann** [comment]
@KrooshalUX @ashwin-pc @ahopp can you take a look at this? What are next steps?

**Comment by KrooshalUX** [comment]
@seanneumann  we are moving away from the E in favor of the new experimental guidance. We have updated OUI to include new experimental badges and messaging will be corrected there. We will need to create a separate issues to have each product change their experimental UI to the new guidance and new OUIExperimentalBadge component. I would suggest closing this out as a wont-do.
